### DISCUSSION TITLE
Is it possible to deploy to the same domain from multiple feature branches?

### DISCUSSION THREAD

**Discussion by iamrakasi** [vercel/vercel#8296] [General]
The UI only allows one branch to be selected but I want multiple feature branches to be deployed to a single "unstable" domain.
![image](https://user-images.githubusercontent.com/48876043/182500728-3816973b-b7ba-4a6c-8512-998df98475c2.png)

**Comment by leerob** [comment]
Could you elaborate more? How can two branches point to the same domain? Is it a subdomain?

**Reply by iamrakasi** [reply]
Yes. For example I have 2 branches: `feature-1` and `feature-2`. When I check-in my changes to either one of those branches I want it to be deployed to `unstable.mydomain.com`. 
I understand there is a chance for overwrite, but it doesn't matter for my use case. My backend doesn't scale well with unknown number of domains, so one `unstable` domain works.
As of now the cumbersome alternative is to change the branch in Vercel domain settings every time I check-in to one of my feature branches.

**Reply by amyegan** [reply]
We don't have a way to assign a domain to more than one branch because of the risk of errors and instability with that approach. Most people use a "staging" branch and merge their feature branches into that branch. You can learn more about setting up a staging environment here https://vercel.com/support/articles/set-up-a-staging-environment-on-vercel
